src/helpers/slack.ts

```diff
@@ -141,15 +141,21 @@ export async function addReactionToSlackPost(
 export async function postMessageToSlack(
     slackWebClient: SlackWebClient,
     text: string,
-    slackChannel: string
+    slackChannel: string,
+    thread_ts?: string
 ) {
-    const postMessageResponse = await slackWebClient.chat.postMessage({
-        text: text,
-        channel: slackChannel,
-    });
-    console.log(
-        `Successfully sent message ${postMessageResponse.ts} to slack channel ${slackChannel}`
-    );
+    if (thread_ts) {
+        return await slackWebClient.chat.postMessage({
+            text: text,
+            channel: slackChannel,
+            thread_ts: thread_ts,
+        });
+    } else {
+        return await slackWebClient.chat.postMessage({
+            text: text,
+            channel: slackChannel,
+        });
+    }
```

Comment:
```suggestion
    let message = {
      text,
      channel: slackChannel
    };
    
    if (threadTs) { //for consistency make it camelCase like slackChannel :)
      message.thread_ts = threadTs;
    }
    
        return await slackWebClient.chat.postMessage(message);
    }
```

Else can be avoided 90% of the time :).

Reply:
Thank you @geshan , I did remove the else block 😍 

I also added a type to message var.